src/components/CampaignInteractionStepsForm.jsx

```diff
@@ -50,16 +43,31 @@ export default class CampaignInteractionStepsForm extends React.Component {
     interactionSteps: this.props.formValues.interactionSteps[0]
       ? this.props.formValues.interactionSteps
       : [
-          {
-            id: "newId",
-            parentInteractionId: null,
-            questionText: "",
-            answerOption: "",
-            script: "",
-            answerActions: "",
-            isDeleted: false
-          }
-        ]
+        {
+          id: "newId",
+          parentInteractionId: null,
+          questionText: "",
+          answerOption: "",
+          script: "",
+          answerActions: "",
+          isDeleted: false
+        }
+      ],
+    displayAllSteps: false
+  };
+
+  /*
+    this.statye.displayAllSteps is used to cause only the root interaction
+    node to render when the component first mounts.  ComponentDidMount sets
+    displayAllSteps to true, which forces render to run again, this time
+    including all interaction steps. This cuts half the time required to
+    render the interaction steps after clicking the header to expand the
+    interaction steps card.
+  */
```

Comment:
can we add a 'FUTURE: Re-evaluate after React 16 upgrade' comment

Reply:
@lperson I merged this into `stage-main` but maybe you could even put this comment into a future PR that's unrelated?

Reply:
#1342